sklearn_extra/cluster/_k_medoids.py

```diff
@@ -324,7 +324,13 @@ def transform(self, X):
             check_is_fitted(self, "cluster_centers_")
 
             Y = self.cluster_centers_
-            return pairwise_distances(X, Y=Y, metric=self.metric)
+            if self.metric == "seuclidean":
+                V = np.var(np.vstack([X, Y]), axis=0, ddof=1)
+                DXY = pairwise_distances(X, Y=Y, metric=self.metric, V=V)
+            else:
+                DXY = pairwise_distances(X, Y=Y, metric=self.metric)
```

Comment:
Maybe something like the following would be simpler,
```suggestion
            kwargs = {}
            if self.metric == "seuclidean":
                kwargs['V'] = np.var(np.vstack([X, Y]), axis=0, ddof=1)
            DXY = pairwise_distances(X, Y=Y, metric=self.metric, **kwargs)
```
(and same below)